probes/securityPolicyContainsDisclosure/def.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: securityPolicyContainsDisclosure
+short: Check that the security policy indicates a vulnerability disclosure process.
+motivation: >
+  If someone finds a vulnerability in the project, it is important for them to be able to communicate it to the maintainers.
+implementation: >
+  The implementation looks for strings "Disclos" and "Vuln".
+outcome:
+  - If information about the disclosure process is found in the security policy file, the probe one finding with OutcomePositive (1).
+  - If no information about the disclosure process is found, the probe returns one finding with OutcomeNegative (0).
+  - if not file is present, one finding with OutcomeNegative (0).
+remediation:
```

Comment:
The step-by-step guide contains a list of things to write in the security policy. I hope that if someone follows the guide, they'll pass the check. wdut?

Reply:
I think this is a technical writing question. 

We benefit from keeping remediation info limited to the test at hand, instead of writing a general one for all security policy probes. And my guess is that someone could write a security policy following those instructions in a way where the probe still fails

Reply:
agreed. Let me simplify each of the remediation steps then.

Reply:
Created https://github.com/ossf/scorecard/issues/3323 or tracking. Want Kara to give some advice.